### PR TITLE
Fix #649 ('reopen' from PR #119)

### PR DESCRIPTION
**Pull request by IsaTab**

### CODE REVIEW

**Comment by Miguelcldn** [comment]
Double-rendering problem, waiting for the fix

File: clair/src/developer.js [hunk 1]
```diff
@@ -436,29 +436,49 @@ function Developer (){
...
-                .easing(TWEEN.Easing.Exponential.InOut)
-                .start();
-
+                var move = dragTile(mesh, section, center).start();
+                window.headers.status[tile.platformID] = 'on';
                 section += 1;
             }
         }
+
+        new TWEEN.Tween(this)
```
Comment (Miguelcldn) [inline clair/src/developer.js]: Not needed anymore, please delete this empty Tween

File: clair/src/tileManager.js [hunk 2]
```diff
@@ -652,13 +641,12 @@ function TileManager() {
...
 
-
-        new TWEEN.Tween(this)
-            .to({}, duration * 2 + self.elementsByGroup * DELAY)
-            .onUpdate(render)
-            .start();
-
-        setTimeout(window.screenshotsAndroid.show, duration);
+        if(animation !== undefined){
+            new TWEEN.Tween(this)
```
Comment (Miguelcldn) [inline clair/src/tileManager.js]: Delete this empty Tween too

File: clair/src/tileManager.js [hunk 3]
```diff
@@ -978,4 +968,54 @@ function TileManager() {
...
+            var index = self.elementsByGroup[id][j];
+            var target = window.helper.getSpecificTile(index);
+            var animate;
+            if(visibility === 'show')
+                animate = noitamina(target.mesh, target.target.show, delay).start();
+            else
+                animate = noitamina(target.mesh, target.target.hide, delay).start();
+        }
+
+        new TWEEN.Tween(this)
```
Comment (Miguelcldn) [inline clair/src/tileManager.js]: This one too

**Comment by IsaTab** [comment]
Removed empty tweens as requested, please test the effects of this change on the objects animation (specifically while showing/hiding a group's tiles).

**Comment by Miguelcldn** [comment]
I'm going to close it until you finish the merge, in order to get a production-ready version

**Pull request closed without merging**